ktaglib/.cxx/cmake/debug/arm64-v8a/android_gradle_build_targets.stdout.txt

```diff
@@ -0,0 +1,119 @@
+ninja: Entering directory `/home/somal/GSoCProgress/MuBandroidnewtry/musicbrainz-android/ktaglib/.cxx/cmake/debug/arm64-v8a'
```

Comment:
Maybe this `stdout.txt` is not needed to be commited?

Reply:
no they are not required.
I will make the changes